### ISSUE TITLE
Should this repository include container-related files?

### ISSUE THREAD

**Issue by colisee** [LibreBooking/app#129]
Hi,

Is it right to include container files (ex: Dockerfile, docker-compose.yml) in this repository?

Shouldn't this repository solely focus on the application itself and have maybe another repository that takes care of the containerization aspects, as it is done with other applications, such as [Nextcloud](https://github.com/nextcloud/docker) ? For your information, I have created repository [docker-librebooking](https://github.com/colisee/docker-librebooking) to illustrate this point.

Kind regards.

Robin

**Comment by effgarces** [comment]
I agree with that approach, it's alot cleaner and will make things simpler for the users. I'm going to take a better look at your repoand will do my best to create a repository just for that.

Thank you for the suggestion and example

**Comment by colisee** [comment]
May I then suggest that:

1. You create a GitHub organization called librebooking
2. You transfer your repo to the organization. It could be renamed “app”
3. I transfer my repo to the organization and rename it as “docker”
4. You remove at one point the docker-related files from the “app” repo

Would that be suitable?

**Comment by effgarces** [comment]
Thanks for the sugestion and especially for listing the steps involved. I had to take some time off to recover and I'm only now returning to work, so I will be slowly getting back to librebooking and try to be more active.

**Comment by colisee** [comment]
My fault, should have closed this issue long time ago!

**Comment by JohnVillalovos** [comment]
Thanks @colisee   I'll go ahead and close it.
